core/src/main/scala/com/avsystem/commons/SharedExtensions.scala

```diff
@@ -26,6 +25,8 @@ trait SharedExtensions {
 
   implicit def futureCompanionOps(fut: Future.type): FutureCompanionOps.type = FutureCompanionOps
 
+  implicit def lazyOptOps[A](opt: => Opt[A]): LazyOptOps[A] = new LazyOptOps(() => opt)
```

Comment:
You don't need this in `SharedExtensions` since `Opt` is a type we control. The compiler will find the implicit if it's in `Opt` companion just as well.